src/party_list_menu.c

```diff
@@ -28,10 +28,6 @@
 
 // TODO: Clean this
```

Comment:
Is this TODO still relevant?

Reply:
I have no idea what this TODO referred to. LOL. But I can remove it.